### ISSUE TITLE
Install without npm. Use *.alfredworkflow file method.

### ISSUE THREAD

**Issue by michaelwoodruff** [jonrohan/alfred-figma#5]
Install without npm? Use *.alfredworkflow file method.

I can't get npm to work. It may be corporate laptop permission issue.

**Comment by megaroeny** [comment]
Agreed. You should provide the `.alfredworkflow` file in the Releases 🙏 

Providing clearer/more details documentation configuration would be appreciated too. Some of (like myself) might not be familiar with creating files with access tokens, etc.

**Comment by megaroeny** [comment]
A follow-up to this — There seems to be an issue now with alfred-link (or alfred-npm), so those of us trying to use this method can't even install Alfred-Figma ☹️

See my comment here: https://github.com/SamVerschueren/alfred-link/pull/20#issuecomment-472024948

**Comment by jonrohan** [comment]
I’d love to wire this up automatically. Keeping open until I can get around to it

**Comment by megaroeny** [comment]
@jonrohan was this updated to the latest version of Alfy now? Did it fix the issue with installing via npm? I saw some commits around that

**Comment by jonrohan** [comment]
It was updated in #6 , I completely forgot to publish it. v 0.3.0 is out. I hope that fixes the npm install issue

**Comment by megaroeny** [comment]
@jonrohan I tried running the installation command again, but ran into this error again ☹️ 
![image](https://user-images.githubusercontent.com/1559447/55734909-639f8d00-59ee-11e9-9450-423c67ad3f09.png)

Am I missing something?

**Comment by MrBeardedGuy** [comment]
Same here. Would it be possible to have a `.alfredworkflow file`???

**Comment by ivanbabko** [comment]
Same error :(

**Comment by megaroeny** [comment]
I'm not going to bother with this anymore since Figma now has a much-improved file browser (with search) and plugins are a thing too. So I would explore that instead. If you ever make an actual workflow file, I'll be excited.

**Comment by banar** [comment]
Unfortunately, the same error.
